### PR TITLE
katana: add page

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/katana.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Crawl a [u]RL or a list of URLs:
+
+`katana -list {{https://example.com,https://google.com,...}}`
+
+- Crawl using [h]ead[l]ess mode using Chromium:
+
+`katana -u {{https://example.com}} -headless`
+
+- Use [p]a[s]sive sources (waybackarchive, commoncrawl, and alienvault) for URL discovery:
```
Comment (kbdharun) [inline pages/common/katana.md:14]:
```suggestion
- Use [p]a[s]sive sources (Wayback Machine, Common Crawl, and alienvault) for URL discovery:
```
Shouldn't this be the Wayback machine or is it something else?

**Review by kbdharun — APPROVED** [review]

File: pages/common/katana.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# katana
+
+> A next-generation crawling and spidering tool.
+> More information: <https://github.com/projectdiscovery/katana>.
+
+- Crawl a [u]RL or a list of URLs:
+
+`katana -list {{https://example.com,https://google.com,...}}`
+
+- Crawl using [h]ead[l]ess mode using Chromium:
```
Comment (gutjuri) [inline pages/common/katana.md:10]:
```suggestion
- Crawl using headless mode using Chromium:
```

IMO it is non-standard that short options can be more than one character, so I'd just leave it out.

File: pages/common/katana.md (hunk 1 above)
Comment (gutjuri) [inline pages/common/katana.md:14]:
```suggestion
- Use passive sources (waybackarchive, commoncrawl, and alienvault) for URL discovery:
```

File: pages/common/katana.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Use [p]a[s]sive sources (waybackarchive, commoncrawl, and alienvault) for URL discovery:
+
+`cat {{example.com}} | katana -passive`
+
+- Pass requests through a proxy (http/socks5) and use custom [H]eaders from a file:
+
+`katana -proxy {{http://127.0.0.1:8080}} -headers {{path/to/headers.txt}} -u {{https://example.com}}`
+
+- Specify crawling [s]trategy, [d]epth of subdirectories to crawl, and rate limiting (requests per second):
```
Comment (gutjuri) [inline pages/common/katana.md:22]:
```suggestion
- Specify the crawling [s]trategy, [d]epth of subdirectories to crawl, and rate limiting (requests per second):
```

**Review by gutjuri — COMMENTED** [review]
Thanks, I have some suggestions.

File: pages/common/katana.md (hunk 2 above)
Comment (fazlearefin) [reply pages/common/katana.md:10]: @gutjuri This practice of using multi chars for short options exists in other places, e.g. `nuclei`, `httpx`

Comment (vitorhcl) [reply pages/common/katana.md:10]: It's a different case there, it's because they are for abbreviated options. We only use this when the long options is abbreviated or when there is a long option with these letters.

Comment (fazlearefin) [reply pages/common/katana.md:10]: I am finding it hard to follow. I think we should clarify the style guide. Anyway, I will accept the suggestion by gutjuri.

File: pages/common/katana.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# katana
+
+> A next-generation crawling and spidering tool.
+> More information: <https://github.com/projectdiscovery/katana>.
+
+- Crawl a [u]RL or a list of URLs:
```
Comment (vitorhcl) [inline pages/common/katana.md:6]:
```suggestion
- Crawl a list of URLs:
```

File: pages/common/katana.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`katana -list {{https://example.com,https://google.com,...}}`
+
+- Crawl using headless mode using Chromium:
```
Comment (vitorhcl) [inline pages/common/katana.md:10]:
```suggestion
- Crawl a [u]RL using headless mode using Chromium:
```

**Review by vitorhcl — COMMENTED** [review]
The mnemonic inthe first example was a bit confusing IMO

**Comment by fazlearefin** [comment]
> The mnemonic in the first example was a bit confusing IMO

Ah yes. My bad. Thanks for fixing.

**Review by kbdharun — APPROVED** [review]

File: pages/common/katana.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# katana
+
+> A next-generation crawling and spidering tool.
```
Comment (vitorhcl) [inline pages/common/katana.md:3]:
```suggestion
> A fast crawler focused on execution in automation pipelines offering both headless and non-headless crawling.
> See also: `gau`, `scrapy`, `waymore`.
```
- "web crawling" is a synonymous for "web spidering", right? I took this description from the usage section of their README
- I added a "See also" section for some similar tools

**Review by vitorhcl — APPROVED** [review]
LGTM after my suggestion

File: pages/common/katana.md (hunk 6 above)
Comment (fazlearefin) [reply pages/common/katana.md:3]: My description was taken from the About section (top right) on the project's GitHub page. I am ok with your suggestion.
